Source/WTF/wtf/ASCIICType.h

```diff
@@ -49,7 +49,9 @@ template<typename CharacterType> constexpr bool isASCIIHexDigit(CharacterType);
 template<typename CharacterType> constexpr bool isASCIILower(CharacterType);
 template<typename CharacterType> constexpr bool isASCIIOctalDigit(CharacterType);
 template<typename CharacterType> constexpr bool isASCIIPrintable(CharacterType);
-template<typename CharacterType> constexpr bool isASCIISpace(CharacterType);
+template<typename CharacterType> constexpr bool isASCIIWhitespace(CharacterType);
+template<typename CharacterType> constexpr bool isNotASCIIWhitespace(CharacterType);
```

Comment:
I suggest putting this in a separate paragraph and documenting why we have an `isNotASCIIWhitespace` function: It’s common enough to want this as a predicate that we include it.